lib/tlaw/namespace.rb

```diff
@@ -139,21 +141,19 @@ def inspect
     end
 
     def describe
-      self.class
-          .describe("#{symbol}(#{param_set.to_hash_code(@parent_params)})")
+      self.class.describe("#{symbol}(#{param_set.to_hash_code(@parent_params)})")
     end
 
     private
 
     def child(symbol, expected_class, **params)
       children[symbol]
-        .tap { |child_class|
+        .tap do |child_class|
```

Comment:
TBH, I prefer curly braces for "functional" method chains. I will not fight for it furiously, but if it is not a big deal for you, 
```ruby
tap {
...
}.new(...)
```
...looks more natural for me, even in huge multiline chains.

Reply:
@zverok I've adopted the `do` blocks for multi-line because many other projects have Rubocops that require this style. I've come to prefer it, but will avoid changing them in this project for now :-)